### ISSUE TITLE
Refactor: Put DB on hold. Add touch support.

### ISSUE THREAD

**Issue by warrendugan** [warrendugan/LoCoLuvRadio#5]
Put DB implementation on hold until ready to scale and serve DB online using Heroku PG. 
Waiting on approval for soundcloud API key. 

- [x] Remove DB implementation //Comment out
- [x] Set up listen component with playlist iframe
- [x] Set up touch support
  - [x] Use ontouchstart jsx equivalent
